fbw-common/src/systems/instruments/src/EFB/Navigation/Navigation.tsx

```diff
@@ -508,14 +530,14 @@ export const ChartViewer = () => {
                 >
                   {chartLinks && provider === 'NAVIGRAPH' && (
                     <p className="text-theme-highlight absolute left-0 top-0 -translate-y-full whitespace-nowrap font-bold transition duration-100">
-                      This chart is linked to {userName}
+                      This chart is linked to {navigraphAuth.user?.preferred_username ?? '<not logged in>'}
```

Comment:
```suggestion
                      This chart is linked to {navigraphAuth.user?.preferred_username ?? '<not logged in>'}
```

As this is displayed to the user perhaps could be localized at some point. (potential TODO for future, maybe not this PR)

Reply:
Agreed. Let's note this down for v4

Reply:
OTOH the charts are all in english.